Libraries/CustomComponents/ListView/ListView.js

```diff
@@ -229,6 +223,14 @@ var ListView = React.createClass({
      */
     removeClippedSubviews: React.PropTypes.bool,
     /**
+     * Makes the sections headers sticky.  The sticky behavior means that it
+     * will scroll with the content at the top of the section until it reaches
+     * the top of the screen, at which point it will stick to the top until it
+     * is pushed off the screen by the next section header.
+     * @platform ios
+     */
+    stickySectionHeaders: React.PropTypes.bool,
```

Comment:
I just checked the code (sorry for not doing it before writing my previous comment): it seems you can flag `stickySectionHeadersEnabled` (as @satya164 suggested) on line 419 and increment an index only if it's enabled. Then 479-484 won't be needed.